app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -51,11 +54,25 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
     : InPersonLocationPostOfficeSearchStep;
 
   // Define different states to be used in human readable array declaration
-  const documentFormStep: FormStep = {
-    name: 'documents',
+  const documentAndSelfieFormStep: FormStep = {
+    name: 'documentsAndSelfie',
     form: DocumentsAndSelfieStep,
     title: t('doc_auth.headings.document_capture'),
   };
+  const documentFormStep: FormStep = {
+    name: 'documents',
+    form: DocumentsStep,
+    title: t('doc_auth.headings.document_capture'), // might want to change title to isolated doc capture heading
+  };
+  const selfieFormStep: FormStep = {
+    name: 'selfies',
```

Comment:
i know the documentForm step is named documents ... which contains 2 artifacts but i think think could just be called `selfie`